### DISCUSSION TITLE
k8s cluster restart prometheus

### DISCUSSION THREAD

**Discussion by bizy01** [prometheus/prometheus#9708] [Q&A]
**What did you do?**
in k8s cluster restart prom deployment, new service is not run, but old server is runing 

show log

level=info ts=2021-11-08T12:32:52.769Z caller=repair.go:57 component=tsdb msg="Found healthy block" mint=1636369200113 maxt=1636372800000 ulid=01FKZRDPMZNCPXFXNHSG14K1M7
level=warn ts=2021-11-08T12:32:52.769Z caller=db.go:683 component=tsdb msg="A TSDB lockfile from a previous execution already existed. It was replaced" file=/prometheus-data/lock
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:695 msg="Stopping scrape discovery manager..."
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:709 msg="Stopping notify discovery manager..."
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:731 msg="Stopping scrape manager..."
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:705 msg="Notify discovery manager stopped"
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:725 msg="Scrape manager stopped"
level=info ts=2021-11-08T12:32:52.769Z caller=manager.go:936 component="rule manager" msg="Stopping rule manager..."
level=info ts=2021-11-08T12:32:52.769Z caller=manager.go:946 component="rule manager" msg="Rule manager stopped"
level=info ts=2021-11-08T12:32:52.769Z caller=notifier.go:601 component=notifier msg="Stopping notification manager..."
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:906 msg="Notifier manager stopped"
level=info ts=2021-11-08T12:32:52.769Z caller=main.go:691 msg="Scrape discovery manager stopped"
level=info ts=2021-11-08T12:32:52.771Z caller=tls_config.go:191 component=web msg="TLS is disabled." http2=false
level=error ts=2021-11-08T12:32:52.771Z caller=main.go:915 err="opening storage failed: lock DB directory: resource temporarily unavailable"

**Comment by kirankucharkanti** [comment]
Is this issue is resolved?
